### ISSUE TITLE
[Bug]: Bottles won't manage Steam prefixes

### ISSUE THREAD

**Issue by ThatOverPowered** [bottlesdevs/Bottles#3385]
### Describe the bug

When launching Bottles with "Steam Proton manager" feature, Bottles won't list any installed game via Steam.
I'm using Bottles and Steam both in the Flatpak version.

Bottles [has access](https://docs.usebottles.com/flatpak/cant-enable-steam-proton-manager) to the Steam path.

### To Reproduce

1. Start Bottles.
2. Enable Steam Proton Manager
3. Restart Bottles
4. No games are listed.

### Package

Flatpak from Flathub

### Distribution

Arch Linux

### Debugging Information

```shell
Official Package: true
Version: '51.11'
DE/WM: gnome
Display:
    X.org: true
    X.org (port): :0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.9.3-zen1-1-zen
Disk:
    Total: 16777310208
    Free: 16777146368
RAM:
    MemTotal: 31.3GiB
    MemAvailable: 26.8GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/thatoverpowered/.local/share/flatpak
F: Opening user flatpak installation at path /home/thatoverpowered/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/thatoverpowered/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: /var/lib/flatpak/runtime/org.gnome.Platform/x86_64/45/99d562071771cdcfe51f9afa912f262cfb8eb83258d83d827f4caeedd4f273ea/files/lib32 does not exist
F: Cleaning up unused container id 1220992146
F: Cleaning up per-app-ID state for com.usebottles.bottles
F: Allocated instance id 2374227690
F: Add defaults in dir /com/usebottles/bottles/
F: Add locks in dir /com/usebottles/bottles/
F: Allowing wayland access
F: Allowing x11 access
F: Allowing pulseaudio access
F: Pulseaudio user configuration file '/home/thatoverpowered/.config/pulse/client.conf': Error al abrir el archivo /home/thatoverpowered/.config/pulse/client.conf: No existe el fichero o el directorio
F: Running 'bwrap --args 44 -- xdg-dbus-proxy --args=46'
F: Running 'bwrap --args 44 -- bottles'
17:36:00 (INFO) Bottles Started! 
17:36:01 (INFO) Performing Bottles checks… 
17:36:01 (INFO) Steam path doesn't exist, creating now. 
17:36:01 (INFO) Dxvks found:
	 - dxvk-2.3.1
 
17:36:01 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.12
 
17:36:01 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.7.0
 
17:36:01 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1
 
17:36:01 (INFO) Runners found:
	 - soda-9.0-1
	 - sys-wine-9.0
 
17:36:01 (WARNING) A Steam prefix was found, but it is a Proton one: 1493710, skipping… 
17:36:01 (WARNING) A Steam prefix was found, but it is a Proton one: 1493710, skipping… 
17:36:02 (INFO) Catalog installers loaded 
17:36:02 (INFO) Catalog dependencies loaded 
17:36:02 (INFO) Catalog components loaded
```

### Additional context

![imagen](https://github.com/bottlesdevs/Bottles/assets/50776804/9fe4f136-e04c-4afc-8e7c-be1a26e7a3da)
![imagen](https://github.com/bottlesdevs/Bottles/assets/50776804/8c4ee7c6-1a69-4706-b7e7-ff507d75892f)
![imagen](https://github.com/bottlesdevs/Bottles/assets/50776804/1ca1c7e0-7eea-4ae0-bac2-5226a87994a5)

**Comment by ThatOverPowered** [comment]
I believe this should be mentioned in the documentation.

In Steam, whether Flatpak or not, when installing a game to a disk other than the default (where Steam itself is installed), Bottles must be given access to that location, or at least, access to the folder containing the game (default SteamLibrary), otherwise, no matter how much Bottles is given permission to `xdg-data/Steam` or `~/.var/app/com.valvesoftware.Steam/data/Steam`, Bottles will not be able to read the wine prefix.

![imagen](https://github.com/bottlesdevs/Bottles/assets/50776804/0829ad2b-f4c5-41d5-b8f8-b9635053c83c)
![imagen](https://github.com/bottlesdevs/Bottles/assets/50776804/6872cdf9-9529-45c6-b78d-6ec83a3e12eb)
![imagen](https://github.com/bottlesdevs/Bottles/assets/50776804/29205ce5-a350-46fd-843a-891abf2af872)

**Comment by xioren** [comment]
Yes this is mostly a quirk of Flatpak sandboxing, not Bottles. But indeed it might be useful to mention it in the documentation. Pull requests welcome.

**Comment by Neon-44** [comment]
I have it on the same Disk. The Problem is that some are installed into ~/.var/app/com.valvesoftware.Steam/data/Steam, in which case they are detected correctly, but some are installed into ~/.var/app/com.valvesoftware.Steam/.local/share/Steam, in which case they aren't detected, even if i put that path into flatseal

**Comment by ykmlg** [comment]
I have the same issue, only that even after adding the disk and giving permission to Steamapps on that disk in Flatseal it still doesn't show the prefix in Bottles.

**Comment by Neon-44** [comment]
I just realized I never posted how I fixed this and by now I forgot.

I think I had to delete the old Proton prefixes Folder in Steam? in Steam/steamapps/common?

I think it somehow didn't recognize the one with the "(beta)" in it?

**Comment by ykmlg** [comment]
ok so adding steam prefixes from different drives only works by adding the path to "Steamapps" that is on the other drive and restarting your pc. if it doesn't work, most of the time deleting the prefixes and letting steam make a new one fixes it. that's how I got Squad to work.

**Comment by gregthwuen** [comment]
I’m encountering a similar issue that might be related: Bottles isn’t listing the prefixes for games installed on an external drive, despite having all the necessary permissions. It appears that Bottles expects the prefixes to be on the same drive as the game itself. However, by default, Steam creates prefixes only in the `compatdata` directory on the main drive.

As a temporary workaround, I’ve created a symlink in the external drive's `steamapps` directory that points to the `compatdata` directory on the main drive.

**Comment by qwrwed** [comment]
I'm having the same issue on Steam Deck with a separate games partition - I'm also using a symlink in the external drive's `steamapps` directory that points to the `compatdata` directory on the main drive, have given Bottles permissions to the folder paths mentioned in this and similar threads, and have tried deleting and re-creating/installing the compatdata folder and the game itself. 

But still only a small amount of my games are showing in the Steam Proton section, and the rest are not - there doesn't seem to be any pattern or consistency as to which games show up and which ones don't.

**Comment by stashymane** [comment]
Same here, I only got CS2 to show up at some point, now nothing does, not even CS2. I gave Bottles access to every single SteamLibrary I have and the base Steam folder. Steam is installed as a dnf package. Tried removing all `compatdata` folders, tried resetting bottles, it keeps returning `Steam path doesn't exist, creating now`.

**Comment by Nekkowe** [comment]
It still persists, I'm having that same issue right now. 
Flatpak Bottles, Native Steam, all necessary permissions granted, it allows me to flip the toggle to "On", but the list of bottles only shows my regular bottles (not even the headlines that I've seen in some other peoples' screenshots):

<img width="867" height="599" alt="Image" src="https://github.com/user-attachments/assets/06ab83e3-d3fd-4d79-9932-59f89a09223a" />

<img width="634" height="686" alt="Image" src="https://github.com/user-attachments/assets/d2bbaf45-1901-4edc-88a3-752091d75ea8" />

**Comment by EoinGohery** [comment]
I am seeing this same issue, just as the comment above mentioned. Any more solutions to this?

**Comment by electroniumez** [comment]
I have this issue but not for steam games prefixs
can bottle show non steam games prefixs too?
